acquire/acquire.py

```diff
@@ -1124,12 +1124,20 @@ class History(Module):
                 "Local Settings/Application Data/Google/Chrom*/User Data",
                 # Chromium - RHEL/Ubuntu - snap
                 "snap/chromium/common/chromium",
+                # Brave - Windows
+                "AppData/Local/BraveSoftware/Brave-Browser/User Data/Default",
+                "AppData/Roaming/BraveSoftware/Brave-Browser/User Data/Default",
+                # Brave - Linux
+                ".config/BraveSoftware/Default",
+                # Brave - MacOS
+                "Library/Application Support/BraveSoftware/Default",
```

Comment:
It will look for all the available profiles (including `Default`), so you should leave that out ;) 

```suggestion
                # Brave - Windows
                "AppData/Local/BraveSoftware/Brave-Browser/User Data",
                "AppData/Roaming/BraveSoftware/Brave-Browser/User Data",
                # Brave - Linux
                ".config/BraveSoftware",
                # Brave - MacOS
                "Library/Application Support/BraveSoftware",
```